README_zh.md

```diff
@@ -1,6 +1,8 @@
 # Standard Go Project Layout
 
+翻译:
```

Comment:
I just translated via google. Please check who know Chinese :-)

Reply:
hopefully somebody will be able to double check it :)